instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -125,6 +128,13 @@ def validated_operation(operation)
             operation if PG::Constants::SQL_COMMANDS.include?(operation)
           end
 
+          def collection_name(text)
+            # Capture the first word (including letters, digits, underscores, & '.', ) that follows common table commands
+            pattern = /\b(?:FROM|INTO|UPDATE|CREATE\s+TABLE(?:\s+IF\s+NOT\s+EXISTS)?|DROP\s+TABLE\s+IF\s+EXISTS)\s+([\w\.]+)/i
```

Comment:
In today's SIG meeting we had some ideas:

* to avoid local variable assignment, could this regex be a frozen constant in the class?
* peek at what other PG (or other SQLy db?) instrumentation (Ruby or in other language OTel instrumentation?) are doing to pluck out connection/table name from queries when the client doesn't know

Reply:
- [Moved](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1087/commits/7bb9db0ac92325469a56682e1ce84ef6dc11db8d#diff-083a2dca6ccc4e55e5bb30b6cd3a69016d1098bcd3db4a7286753ac410324134R18) to a constant ◡̈ Looks like we didn't need to freeze it - immutable anyways. 
- No such luck yet in finding other instrumentations, will look more today!

Reply:
I'm not finding anything in other instrumentations - I feel like someone would've solved this already but maybe not 🤷‍♀️